packages/blockchain-wallet-v4-frontend/src/scenes/app.js

```diff
@@ -43,7 +43,7 @@ class App extends React.PureComponent {
           <ThemeProvider>
             <ConnectedRouter history={history}>
               <Switch>
-                <PublicLayout path='/a/:payload' component={Goals} />
+                <PublicLayout path='/open/:payload' component={Goals} />
```

Comment:
Will this break `open` on line 49?

Reply:
Everything seems to be working but I did not realise we have some duplicated functionality here. Goals and Open seem to be doing the same thing except Open is slightly more generic. I think we could get rid of either scenes/Goals or scenes/Open

Reply:
Lets get rid of scenes/Open please since it's actually not used at all anymore. Thanks!